build.sh

```diff
@@ -46,6 +46,9 @@ elif [ "$target" == "mkrelease" ]; then
     $boarddir/mkimage.sh
     cp $outputdir/images/$osname-$board.img $basedir
     mv $basedir/$osname-$board.img  $basedir/$osname-$board-$osversion.img
+    rm -f $basedir/$osname-$board-$osversion.img.xz
+    xz -6ek $basedir/$osname-$board-$osversion.img
```

Comment:
Are we sure about `-e`?

Reply:
There's no harm in adding the -e option. It uses slightly more time to compress to get a better compression, at the same memory usage as without -e option. We can get rid of the -e option if you like.